terraform/install.sh

```diff
@@ -174,6 +174,24 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  external_ip="";
+  while [ -z $external_ip ]; do
+     external_ip=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'); 
+     [ -z "$external_ip" ] && sleep 30; 
+  done;
+
+  gcp_monitoring_path="https://console.cloud.google.com/monitoring?project=$project_id"
+  log "Please create a monitoring workspace for the project by clicking on the following link: $gcp_monitoring_path"
+  read -p "When you are done, please press enter to continue"
+
```

Comment:
Can we add some kind of check here to make sure it was enabled>

Reply:
We should add a tracking task for this for next iterations. Currently the API is in alpha. We won't be able to check without having the project whitelisted, which is impossible in our scenario

Reply:
I will open a task for this - the workaround is to use the Dashboards API since Workspaces API is in EAP. We'll add this in after the release.